### PR TITLE
Further Encyclopedia Revision

### PR DESCRIPTION
**Pull request by Gothyoba**
I initially intended this to just remove the Southwood and add Southport, but I have since found many other small improvements to make. Because of the major lore implication of some of these changes, I am posting this as a draft early. I will also be trying to follow changes to the TSG Re-Revision and update this accordingly over time. This should be merged at least once the TSG Re-Revision is nearly or fully merged into mainline. This will be much smaller than the last revision.

- Added a detail on Mount Morogor being the highest active volcano in Morogor, not just a random volcano. This makes it appear more sensible that it is included.

- Mentioned Kerlath and the Far South at the end of the Wesnoth description.

- Clarified Kerlath is the southernmost province in the Westin description, not just a southern province.

- Added Southport to the Wesnoth section from the TSG Re-Revision.

- Removed the Southwood due to its removal from the TSG Re-Revision.

- Added the Vale of Blossoming Trees to the section on Southwestern Elves.

- Replaced "men" with "humans" in the Northlands description, which I accidentally copied from the old description. I see no reason for this not being gender-neutral. (Actually, these small changes could be merged separately earlier)

- Mentioned Saurgrath in the Northlands.

- Added a description of the Mourned Hills which mentions Lmarig, as suggested by CelticMinstrel

- Mainlined many ideas from this thread with slight changes: https://forums.wesnoth.org/viewtopic.php?t=52368&hilit=al+shizirad&start=45

I haven't played TDG yet, so there might be more changes from that campaign. I don’t Fort Garard is significant enough to become a notable city in the Northlands, but I might mention it in one of the descriptions of something else.

@Dalas121 What are your thoughts on these changes?

### CODE REVIEW

File: data/core/encyclopedia/geography.cfg [hunk 1]
```diff
@@ -170,7 +171,7 @@ Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish
         _ "<header>text='Notable land features:'</header>
     • <ref>dst='heart_mountains' text='Heart Mountains'</ref>: A virtually impassable barrier between the river country and the Northern Plains.
     • Heartfangs: The particularly forbidding stretch of high peaks southwest of Lake Vrug and north of the Forest of Wesmere. The most inhospitable and dangerous portion of the Heart Mountains; only hermits, madmen, and mages live there.
-    • Swamp of Dread: A very large bog located between the Heart Mountains and the Great River. A notoriously dangerous place.
+    • Swamp of Dread: A very large bog located between the Heart Mountains and the Great River. A notoriously dangerous place filled with violent saurians.
```
Comment (CelticMinstrel) [inline data/core/encyclopedia/geography.cfg:174]: I don't think "violent" is a very good word to use to describe saurians…

File: data/core/encyclopedia/geography.cfg [hunk 2]
```diff
@@ -233,12 +236,19 @@ Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish
 [topic]
     id=far_south
     title= _ "Far South"
-    text= _ "A vast region south of Kerlath home to the Dunefolk Civilization. It is also inhabited by many local creatures, including the Jinn and Wyverns." + "
-
+    text= _ "A vast region south of Kerlath home to the Dunefolk Civilization. It is also inhabited by many local creatures, including the Jinn and Wyverns. Very little is known for certain about this land." + "
```
Comment (CelticMinstrel) [inline data/core/encyclopedia/geography.cfg:239]: I just realized that Kerlath is mentioned several times, yet never clearly defined. (It looks like it's implicitly defined in Southwest Elven Lands?)

Comment (Gothyoba) [reply data/core/encyclopedia/geography.cfg:239]: Kerlath is mentioned as a province in Southern Wesnoth in the Westin description, though that is quite vague. I might replace it with the more recognizable Wesnoth. I could also describe it more explicititly in the Wesnoth description as well. (btw thanks for fixing all the indenation and breaking up the strings)

File: data/core/encyclopedia/geography.cfg (hunk 1 above)
Comment (Gothyoba) [reply data/core/encyclopedia/geography.cfg:174]: They are said to be very violent in the unit description, and form the perspective of Wesnoth they would likely be considered quite barbaric, even if the unit description is exaggerated. However, I should probably choose a better adjective to convey that.

**Comment by Gothyoba** [comment]
I’ve reduce the Far South changes to only adding the Ashland Desert. let me know if anyone thinks ptherwise, but Injust don’t think we should be limitjgn the possibilities on the sotry wothout a single proper Dunefolk campaign. I’m still a bit uncertain if I should add one or two cities, but probably not four. While Palmiya is commonly ahown on amps due to appearing on the OoA map, it is rarely used as a setting.

**Comment by Gothyoba** [comment]
@Dalas121 (This is just a reminder of this and what has changed in the PR and why) Sorry for going back on forth on my Far South proposals here over the past few days. I have decided to currently add a large amount of proposals from the thread above, though the list of added content will probably be trimmed over time. (e.g. Impassable Jungle and Jungle of Death are a bit redundant in the context of a lack of other mainlined material in the region). I would like at least your general opinion on whether or not this is way too much, or if this should be what I should be doing. I chose to mainline a lot of it mainly for the positive reception for these proposal back then, but I understand there are good points for not including so much detail, especially when for much of this the encyclopedia would be the only mention of this lore.

Thanks for all your reviews and suggestions!

**Pull request closed without merging**